python_modules/dagster/dagster/components/resolved/context.py

```diff
@@ -93,8 +59,21 @@ def at_path(self, path_part: Union[str, int]):
 
     @staticmethod
     def default(source_position_tree: Optional[SourcePositionTree] = None) -> "ResolutionContext":
+        # Create the automation_condition object for backward compatibility
+        automation_condition_obj = {
+            "eager": AutomationCondition.eager,
+            "on_cron": AutomationCondition.on_cron,
+        }
+
         return ResolutionContext(
-            scope={"env": EnvScope(), "automation_condition": automation_condition_scope()},
+            scope={
+                "env": EnvScope(),
+                "dg": DgScope(),
```

Comment:
Is the thought that everything we provide will be in the `dg` scope and what users create will likely be in the top-level namespace (or their own custom)? I'm conflicted on the `dg` prefix, but I can see the appeal of matching the Python bindings.